ur_robot_driver/src/ros/hardware_interface.cpp

```diff
@@ -104,6 +104,9 @@ bool HardwareInterface::init(ros::NodeHandle& root_nh, ros::NodeHandle& robot_hw
     return false;
   }
 
+  // Enables non_blocking_read mode. Useful when used with combined_robot_hw
```

Comment:
We could improve documentation a bit, here. I would especially note that error output on missing packets are suppressed.

Reply:
How does that look now?

Reply:
How about this? https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/commit/529c711c22bb58167a7c5b195dbffc3dd038b2c9